src/maintainer/knowledge_base.rst

```diff
@@ -1309,6 +1309,78 @@ Noarch generic
 
   add some information on r packages which make heavy use of ``noarch: generic``
 
+.. _multi_output_recipes:
+
+Multi-output recipes
+====================
+
+``conda-build`` has the ability to create multiple package artifacts from a single recipe via the
+``outputs`` section in ``meta.yaml``. This is useful in several scenarios, including:
+
+- Distributing a project (which share the same source code) in separate artifacts. For example:
+
+  - A compiled C++ library and its Python bindings:
+    
+    - `mamba-feedstock <https://github.com/conda-forge/mamba-feedstock/blob/main/recipe/meta.yaml>`__
+  
+  - A runtime library and its headers:
+    
+    - `cpp-opentelemetry-sdk <https://github.com/conda-forge/cpp-opentelemetry-sdk-feedstock/blob/main/recipe/meta.yaml>`__
+
+  - A dynamic library and a static version:
+
+    - `libarchive <https://github.com/conda-forge/libarchive-feedstock/blob/main/recipe/meta.yaml>`__
+
+- Distributing the same project with different sets of dependencies. For example:
```

Comment:
This is also a trick to mimic optional dependencies from Python-land. Maybe we should mentioned that here too?

Reply:
Added! 🦸